ibc-core/ics04-channel/types/src/events/mod.rs

```diff
@@ -32,13 +32,14 @@ const CHANNEL_OPEN_ACK_EVENT: &str = "channel_open_ack";
 const CHANNEL_OPEN_CONFIRM_EVENT: &str = "channel_open_confirm";
 const CHANNEL_CLOSE_INIT_EVENT: &str = "channel_close_init";
 const CHANNEL_CLOSE_CONFIRM_EVENT: &str = "channel_close_confirm";
+const CHANNEL_CLOSED_EVENT: &str = "channel_close";
```

Comment:
Moving the `CHANNEL_CLOSED_EVENT` constant under the channel event types section introduces some ambiguity regarding the semantics and the purpose of the constant. The string itself being just `"channel_close"` when all the other `"channel_*"` constants carry some additional meaning also contributes to the ambiguity. 

Can we at least add a docstring comment on the `CHANNEL_CLOSED_EVENT` constant?

Reply:
you mean `"channel_close_*", right?

Reply:
I would suggest just adding a reference link to `ibc-go`. Because we copy these values from there.

Reply:
[ibc-go](https://github.com/cosmos/ibc-go/blob/c4413c5877f9ef883494da1721cb18caaba7f7f5/modules/core/04-channel/types/events.go#L60) actually uses `channel_close`, not `channel_closed`, which is a bit confusing. But if parity is what we're aiming for, then I guess we go with `channel_close`.

Reply:
Yea, that's what I meant that we don't have a choice here :sweat_smile: